p/scripts/global_view.js

```diff
@@ -73,6 +73,16 @@ function init_close_panel() {
 			document.getElementById('overlay').classList.remove('visible');
 			return false;
 		};
+	document.addEventListener('keydown', ev => {
+		let k = (ev.key.trim() || ev.code).toUpperCase();
+		if (k === 'ESC') {
+			k = 'ESCAPE';
+		}
+		if (k == 'ESCAPE') {
+			document.querySelector('#overlay .close').click();
+		}
+		return false;
+	}
```

Comment:
```suggestion
	});
```
Also there's something weird about the indentation.